### PR TITLE
Add some extra php-fpm configurable config

### PR DESCRIPTION
**Pull request by lopsided**

### CODE REVIEW

File: php-fpm/defaults/main.yml [hunk 1]
```diff
@@ -9,8 +11,11 @@ php_fpm_track_errors: Off
 php_fpm_post_max_size: 8M
 php_fpm_upload_max_filesize: 8M
 php_fpm_mysqlnd_collect_statistics: Off
+php_fpm_opcache_memory_consumption: 128
+php_fpm_opcache_max_accelerated_files: 100000
 php_fpm_opcache_file_cache: ~
 php_fpm_opcache_file_cache_only: 0
+php_fpm_opcache_validate_timestamps: 1
```
[the remark this answers is not in the document]
Comment (lopsided) [reply php-fpm/defaults/main.yml]: I was going with the default value from here: http://php.net/manual/en/opcache.configuration.php

File: php-fpm/templates/php.ini.j2 [hunk 2]
```diff
@@ -161,15 +161,16 @@ ldap.max_links = -1
...
 opcache.interned_strings_buffer = 8
-opcache.max_accelerated_files = 100000
+opcache.max_accelerated_files = {{ php_fpm_opcache_max_accelerated_files }}
 opcache.save_comments = 1
 opcache.fast_shutdown = 1
 opcache.file_cache = "{{ php_fpm_opcache_file_cache }}"
 opcache.file_cache_only = {{ php_fpm_opcache_file_cache_only }}
 opcache.file_cache_consistency_checks = 1
 opcache.file_cache_fallback = 1
+opcache.validate_timestamps = {{ php_fpm_opcache_validate_timestamps }}
```
[the remark this answers is not in the document]
Comment (lopsided) [reply php-fpm/templates/php.ini.j2]: Yep, that's why sf performance tweaks suggest switching it to 0

File: php-fpm/templates/php.ini.j2 [hunk 3]
```diff
@@ -13,8 +13,8 @@ unserialize_callback_func =
 serialize_precision = -1
 disable_functions = pcntl_alarm,pcntl_fork,pcntl_waitpid,pcntl_wait,pcntl_wifexited,pcntl_wifstopped,pcntl_wifsignaled,pcntl_wifcontinued,pcntl_wexitstatus,pcntl_wtermsig,pcntl_wstopsig,pcntl_signal,pcntl_signal_get_handler,pcntl_signal_dispatch,pcntl_get_last_error,pcntl_strerror,pcntl_sigprocmask,pcntl_sigwaitinfo,pcntl_sigtimedwait,pcntl_exec,pcntl_getpriority,pcntl_setpriority,pcntl_async_signals,
 disable_classes =
-realpath_cache_size = 16M
-realpath_cache_ttl = 120
+realpath_cache_size = {{ php_fpm_realpath_cache_size }}
+realpath_cache_ttl = {{ php_fpm_realpath_cache_ttl }}
```
[the remark this answers is not in the document]
Comment (lopsided) [reply php-fpm/templates/php.ini.j2]: http://symfony.com/doc/3.4/performance.html#configure-the-php-realpath-cache

[the remark this answers is not in the document]
Comment (TomAdam) [reply php-fpm/templates/php.ini.j2]: Granted TTL could be higher but I don't think these need set on a project to project basis. Perhaps just set to 16M and 600?

File: php-fpm/templates/php.ini.j2 (hunk 2 above)
[the remark this answers is not in the document]
Comment (TomAdam) [reply php-fpm/templates/php.ini.j2]: Good point

File: php-fpm/templates/php.ini.j2 (hunk 3 above)
[the remark this answers is not in the document]
Comment (TomAdam) [reply php-fpm/templates/php.ini.j2]: No reason not to have them configurable

**Pull request merged**